src/NexusMods.App.UI/Controls/ModInfo/ViewModFiles/ViewModFilesViewModel.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.ObjectModel;
+using DynamicData;
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Games.DTO;
+using NexusMods.Abstractions.IO;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.Loadouts.Files;
+using NexusMods.Abstractions.Loadouts.Mods;
+using NexusMods.App.UI.Controls.Trees.Files;
+using NexusMods.Paths.Trees.Traits;
+using ModFileNode = NexusMods.App.UI.TreeNodeVM<NexusMods.App.UI.Controls.Trees.Files.IFileTreeNodeViewModel, NexusMods.Abstractions.GameLocators.GamePath>;
+namespace NexusMods.App.UI.Controls.ModInfo.ViewModFiles;
+
+public class ViewModFilesViewModel : AViewModel<IViewModFilesViewModel>, IViewModFilesViewModel
```

Comment:
Add `[UsedImplicitly]` attribute.

Reply:
Question:

This ViewModel is used explicitly, why would I want to add the attribute?

Reply:
That works